app/services/submit_to_app_store/prior_authority/firm_office_payload_builder.rb

```diff
@@ -8,13 +8,12 @@ def initialize(application)
       def payload
         @application.firm_office.as_json(only: %i[
                                            name
-                                           account_number
                                            address_line_1
                                            address_line_2
                                            town
                                            postcode
                                            vat_registered
-                                         ])
+                                         ]).merge(account_number: @application.office_code)
```

Comment:
this is fine if the user only has a single office code.. but if they have multiple wouldn't this pre-select the first one on the next screen?

Reply:
So this is the payload on submission to the app store, once the office code has either explicitly or implicitly been chosen. There is no next screen.

Reply:
(Note that at this point we're no longer referring to the user's office codes, we're just putting into the appropriate part of the payload the _application's_ office code)